<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1839</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="RightUpDownLayout">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="ChatLayout">
         <item>
          <widget class="QTabWidget" name="ChatTab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>300</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::North</enum>
           </property>
           <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <property name="documentMode">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="Channel1">
            <attribute name="title">
             <string>Channel 1</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTextEdit" name="ChatHistory1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                </font>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Channel2">
            <attribute name="title">
             <string>Channel 2</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTextEdit" name="ChatHistory2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustIgnored</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:6.6pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins[Fragi] Bonsoir les tchoins&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="overwriteMode">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ChatInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="maxLength">
            <number>32767</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTabWidget" name="ToolsTab">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>325</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="VolumeTab">
         <attribute name="title">
          <string>Volume</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>300</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>914</width>
               <height>298</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(42,42,42);</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="MicTab">
         <attribute name="title">
          <string>Micro</string>
         </attribute>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="ToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionSpeakers"/>
   <addaction name="actionCapture"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/manage.png</normaloff>:/icons/manage.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSpeakers">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/volume.png</normaloff>:/icons/volume.png</iconset>
   </property>
   <property name="text">
    <string>Speakers</string>
   </property>
  </action>
  <action name="actionCapture">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/capture.png</normaloff>:/icons/capture.png</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/connect.png</normaloff>:/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
